mmv1/third_party/terraform/services/composer/resource_composer_environment_test.go.erb

```diff
@@ -1834,6 +1834,11 @@ resource "google_composer_environment" "test" {
           zone = "us-east1-c"
       }
       environment_size = "ENVIRONMENT_SIZE_MEDIUM"
+	  data_retention_config {
```

Comment:
It would be good to add these fields to an update test as well to make sure they can be updated in-place.

Reply:
Done.